packages/veritone-react-common/src/components/MediaInfoPanel/EditMetadataDialog/index.js

```diff
@@ -128,103 +147,121 @@ class EditMetadataDialog extends Component {
+              <IconButton
+                onClick={this.props.onClose}
+                aria-label="Close"
+                classes={{
+                  root: styles.closeButton
+                }}
+              >
+                <Icon className="icon-close-exit" />
+              </IconButton>
+            </DialogTitle>
+            <DialogContent>
+              <TextField
+                autoFocus
+                margin="normal"
+                id="filename"
+                label="Filename"
+                InputLabelProps={{
+                  shrink: true
+                }}
+                value={this.state.filename}
+                onChange={this.onFileNameChange}
+                fullWidth
+                classes={{
+                  root: styles.textInput
+                }}
+              />
+              <TextField
+                margin="normal"
+                id="source"
+                className={styles.sourceSection}
+                label="Source"
+                placeholder="Enter name of media owner or creator"
+                InputLabelProps={{
+                  shrink: true
+                }}
+                value={this.state.source}
+                onChange={this.onSourceChange}
+                fullWidth
+                classes={{
+                  root: styles.textInput
+                }}
+              />
+              <div className={styles.programImagesSection}>
+                <div
+                  className={styles.imageSection}
+                >
+                  <div className={styles.programImageLabel}>
+                    Program Live Image
+                  </div>
+                  <div className={styles.programLiveImageContainer}>
+                    <img className={styles.programLiveImage}
+                      src={
+                        get(this.state, 'programLiveImage.length') ?
+                          this.state.programLiveImage :
+                          "//static.veritone.com/veritone-ui/default-nullstate.svg"
+                      }
+                    />
+                    <div className={styles.imageOverlay}>
+                      <EditIcon classes={{root:styles.editProgramLiveImageIcon}} className="icon-mode_edit2" onClick={this.handleStartPickFiles('programLiveImage')}/>
+                      { !!get(this.state, 'programLiveImage.length') && <DeleteIcon classes={{root:styles.editProgramLiveImageIcon}} className="icon-trashcan" onClick={this.handleRemoveImage('programLiveImage')}/>}
+                    </div>
+                  </div>
+                </div>
+                <div
+                  className={styles.imageSection}
+                >
+                  <div className={styles.programImageLabel}>Program Image</div>
+                  <div className={styles.programImageContainer}>
+                    <img className={styles.programImage}
+                      src={ 
+                        get(this.state, 'programImage.length') ?
```

Comment:
is there any reason you can't do -> this.state.programLiveImage || '//static.veritone.com/veritone-ui/default-nullstate.svg'?